app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,21 @@
+class CommentsController < ApplicationController
+	before_action :authenticate_user!
```

Comment:
Indentation is off. Please make sure that you use `Indent Using Spaces` and tab_size=2